app/services/reinstate_pending_conditions.rb

```diff
@@ -21,7 +21,7 @@ def save
       if valid?
         ActiveRecord::Base.transaction do
           ApplicationStateChange.new(application_choice).reinstate_pending_conditions!
-          UpdateOfferConditions.new(application_choice: application_choice).call
+          application_choice.offer.set_conditions_to_pending
```

Comment:
Similar to the comment above, https://github.com/DFE-Digital/apply-for-teacher-training/pull/4929/files#r649087167